File: building_map_app/web/js/api_config.js
// Google Maps API 설정
const API_CONFIG = {
    // Google Maps API 키 (실제 배포 시에는 환경변수 사용 권장)
    GOOGLE_MAPS_API_KEY: 'AIzaSyDQK8SOOSJsDH8H7HuPbwHNv72MHE_ZpsA',

    // API 키 유효성 검사
    isApiKeyValid: function() {
        return this.GOOGLE_MAPS_API_KEY &&
               this.GOOGLE_MAPS_API_KEY !== 'YOUR_API_KEY_HERE' &&
               this.GOOGLE_MAPS_API_KEY.length > 0;
    }
};

// 전역으로 내보내기
window.API_CONFIG = API_CONFIG;

File: building_map_app/lib/config/api_config.dart
/// API 키와 설정을 관리하는 클래스
class ApiConfig {
  /// Google Maps API 키
  /// 실제 배포 시에는 환경변수나 보안 저장소에서 가져와야 함
  static const String googleMapsApiKey = 'AIzaSyDQK8SOOSJsDH8H7HuPbwHNv72MHE_ZpsA';

  /// 개발/프로덕션 환경 구분
  static const bool isProduction = false;

  /// API 키 유효성 검사
  static bool isApiKeyValid() {
    return googleMapsApiKey.isNotEmpty && googleMapsApiKey != 'YOUR_API_KEY_HERE';
  }
}